colossus/src/main/scala/colossus/core/ServerSettings.scala

```diff
@@ -42,6 +43,7 @@ import scala.concurrent.duration._
   * completes the shutdown.
   */
 case class ServerSettings(
+    addresses: Seq[String] = Seq("0.0.0.0"),
```

Comment:
I think it would be safer to have the empty list as default, then if the list is empty use `new InetSocketAddress(settings.port)`.